### PR TITLE
Add private mode confirmation prompt

### LINKED ISSUE #1112: No profil picture/Blur profil picture
Hi,

Since few days the profile pictures on each conversation are blurry, and the only way to see which conversation is with who is to select it and to look on the right panel.

This happened since the last Caprine's update.

Cheers,

### PR DESCRIPTION
**Pull request by alexandrepa**
Fixes #1112 
Feel free to change the dialog text, it's not my mother tongue 😄

### CODE REVIEW

File: source/util.ts [hunk 1]
```diff
@@ -34,3 +34,15 @@ export function showRestartDialog(message: string): void {
...
+	return (
+		dialog.showMessageBoxSync({
+			message: 'Are you sure you want to hide names and avatars ?',
+			detail: 'You have triggered the function using Ctrl+Shift+N.',
+			buttons: ['Yes', 'No'],
+			defaultId: 0,
+			cancelId: 1
+		}) === 0
+	);
+}
```
Comment (sindresorhus) [inline source/util.ts]: This should not be in `util.js`. This file is for reusable things.

File: source/util.ts [hunk 2]
```diff
@@ -34,3 +34,15 @@ export function showRestartDialog(message: string): void {
 		app.quit();
 	}
 }
+
+export function confirmPrivateModeDialog(): boolean {
+	return (
+		dialog.showMessageBoxSync({
+			message: 'Are you sure you want to hide names and avatars ?',
+			detail: 'You have triggered the function using Ctrl+Shift+N.',
+			buttons: ['Yes', 'No'],
```
Comment (sindresorhus) [inline source/util.ts]:
```suggestion
			buttons: ['Hide', 'Don\'t Hide'],
```

File: source/util.ts [hunk 3]
```diff
@@ -34,3 +34,15 @@ export function showRestartDialog(message: string): void {
 		app.quit();
 	}
 }
+
+export function confirmPrivateModeDialog(): boolean {
+	return (
+		dialog.showMessageBoxSync({
+			message: 'Are you sure you want to hide names and avatars ?',
```
Comment (sindresorhus) [inline source/util.ts]:
```suggestion
			message: 'Are you sure you want to hide names and avatars?',
```

File: source/util.ts [hunk 4]
```diff
@@ -34,3 +34,15 @@ export function showRestartDialog(message: string): void {
...
+			detail: 'You have triggered the function using Ctrl+Shift+N.',
```
Comment (sindresorhus) [inline source/util.ts]: This is not correct. It could have been Command instead of Control.

**Comment by alexandrepa** [comment]
I think there is a conflict between eslint and prettier with single quote.

**Pull request merged**